app/labor/badge_rewarder.rb

```diff
@@ -1,12 +1,19 @@
 module BadgeRewarder
-  YEARS = { 1 => "one", 2 => "two", 3 => "three", 4 => "four", 5 => "five", 6 => "six", 7 => "seven", 8 => "eight", 9 => "nine", 10 => "ten" }.freeze
+  YEARS = { 1 => "one", 2 => "two", 3 => "three" }.freeze
   REPOSITORIES = ["thepracticaldev/dev.to", "thepracticaldev/DEV-ios", "thepracticaldev/DEV-Android"].freeze
 
+  LONGEST_STREAK_WEEKS = 16
+  LONGEST_STREAK_MESSAGE = "16 weeks! You've achieved the longest #{ApplicationConfig['COMMUNITY_NAME']} writing " \
+    "streak possible. This makes you eligible for special quests in the future. Keep up the amazing contributions to" \
+    " our community!".freeze
+
+  MINIMUM_QUALITY = -25
+
   def self.award_yearly_club_badges
     (1..3).each do |i|
       message = "Happy #{ApplicationConfig['COMMUNITY_NAME']} birthday! Can you believe it's been #{i} #{'year'.pluralize(i)} already?!"
       badge = Badge.find_by!(slug: "#{YEARS[i]}-year-club")
-      User.where("created_at < ? AND created_at > ?", i.year.ago, i.year.ago - 2.days).find_each do |user|
+      User.where(created_at: i.year.ago..i.year.ago - 2.days).find_each do |user|
```

Comment:
I switched this to a range, which turns this into a `BETWEEN` query.  There's one difference though: `BETWEEN` is inclusive while the existing query was exclusive. I felt that this wsa ok in this case, but if people prefer that I can change it back. /cc @benhalpern